packages/language-server/src/completion/completions.ts

```diff
@@ -605,8 +609,24 @@ export function getSuggestionForSupportedFields(
           )
         }
       }
+      // engineType
+      if (currentLine.startsWith('engineType')) {
+        const engineTypesCompletion: CompletionItem[] = engineTypes
+        if (isInsideQuotation) {
+          return {
+            items: engineTypesCompletion,
```

Comment:
```suggestion
            items: engineTypes,
```

Reply:
That just follow the pattern of the existing code - let's not change that now.